src/pages/product/[[...product]].tsx

```diff
@@ -32,14 +35,21 @@ const ProductPage: NextPage<InferGetStaticPropsType<typeof getStaticProps>> = ({
   options,
   product,
   reviewHighlights,
-  reviewList,
+  reviewsIoReviewList,
   trustpilotReviewList,
   details,
   policies,
   breadcrumbs
 }) => {
   const { isFallback } = useRouter();
 
+  const productJsonLdProps = useMemo(() => {
```

Comment:
This operation seems like another transform, similar to what is happening in `getStaticProps`. I'd suggest moving this there for consistency, and just rendering JSON-LD component here.